### PR TITLE
Update for sprint 3-3

### PR DESCRIPTION
**Pull request by cnevinc**
We need to backfill "Click private shortcut" from June 1st. 
"Click Toolbar" from Oct 28th
And the other new events from Dec 16th

Is that possible?

### CODE REVIEW

**Comment by cnevinc** [comment]
Close this first since there's a issue need to confirm with Eddie

**Comment by cnevinc** [comment]
Confirmed with the BI team in Taipei. This is good for review.

File: sql/telemetry/rocket_android_events_v1/view.sql [hunk 1]
```diff
@@ -155,32 +153,42 @@ SELECT
...
         WHEN (event_category IN ('action') ) AND (event_method IN ('start') ) AND (event_object IN ('process') ) AND (event_value IN ('tab_swipe') ) THEN 'Rocket -  Start Tab Swipe Process' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('end') ) AND (event_object IN ('process') ) AND (event_value IN ('tab_swipe') ) THEN 'Rocket -  End Tab Swipe Process' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('start') ) AND (event_object IN ('tab_swipe') ) THEN 'Rocket -  Start Tab Swipe' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('end') ) AND (event_object IN ('tab_swipe') ) THEN 'Rocket -  End Tab Swipe' 
-        WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('share|OPEN_IN_BROWSER|reload|back') ) THEN 'Rocket -  Click Toolbar - Share/Reload/Back' 
+        WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('share', 'OPEN_IN_BROWSER', 'reload', 'back') ) THEN 'Rocket -  Click Toolbar - Share/Reload/Back'
```
Comment (lanyliu) [inline sql/telemetry/rocket_android_events_v1/view.sql]: not aligned with spec
https://docs.google.com/spreadsheets/d/1fralUDT5tZWZgfyFvAhbxHK7b1UQz696yL0xPrdu45A/edit#gid=514507148&range=C12

should be
`WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('share')) THEN 'Rocket -  Click Toolbar - Share' `

`WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('open_in_browser') ) THEN 'Rocket -  Click Toolbar - Open in Browser' `

`WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('reload' ) ) THEN 'Rocket -  Click Toolbar - Reload' `

`WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('back' ) ) THEN 'Rocket -  Click Toolbar - Back' `


please kindly check again

Comment (lanyliu) [reply sql/telemetry/rocket_android_events_v1/view.sql]: @cnevinc

**Comment by akkomar** [comment]
Was this superseded by https://github.com/mozilla/bigquery-etl/pull/690?

**Comment by Conjuror** [comment]
> Was this superseded by #690?

I think so. Please help to close the PR.

**Pull request closed without merging**